common/f3d_materials.md

```diff
@@ -0,0 +1,193 @@
+When in 2 cycle mode, `Texel0` and `Texel1` are swapped, and `Texel1` maps to the next pixel in memory.
+:::
+
+## Sources
+
+Each input available to edit will be one of the inputs selected in the combiner tab. Inputs available can be one of following types:
+
+## Textures
+
+Textures map the output of the source image to pixels using UVs. There are several texture types available to use, and many properties which is covered in their own article: (link I guess).
+
+The combiner supports two textures for use, named [Texture 0]{.key_text} and [Texture 1]{.key_text}. You can load up to 4kb into texture memory (TMEM) at once using any type but Color Indexed (CI), which only supports 2kb, the other 2kb is reserved for palettes.
+
+Fast64 will calculate TMEM usage when selecting a texture. If you're texture is too large, you will have to resize it, or use **Large Texture Mode**. When using large textures, you will have to subdivide the mesh so that there are an equal number of faces as TMEM size divisions; this can be done automatically using the **Create Large Texture Mesh** tool which is located in the Fast64 Tools panel in the 3D viewport.
+
+## Geometry Influenced
+
+Some registers vary their color based on the geometry in the viewport. Only the `Shade` and `LOD Fraction` inputs in the combiner do this.
+
+[Shade]{.key_text} changes it's color based on the geometry mode you have selected. If you are using lighting, then the shade color is based on the `Light` and `Ambient` color. The color will vary between the two depending on the direction of your lighting. If you are not using lighting, then the shade color will be based on vertex colors.
+
+Shade alpha has its own quirks. Shade alpha will be vertex colors unless the `fog` geo mode is enabled, then shade alpha will become the fog value. Fog will change from 0 to 1 depending on how far away the mesh triangles are from the camera. The range of travel depends on the fog intensity chosen.
+
+[LOD Fraction]{.key_text} is a variable that determines how far between LOD levels the current texel will be. This value shifts between 0 and 1 as the camera moves farther away from the target texel. Once reaching one, the value will wrap back to zero until the max LOD level is reached. This value is only enabled while using LOD calculations.
+
+## Color Registers
+
+Color registers are RGBA values that are stored in Display Memory (DMEM) and used for combiner operations. These registers hold a discrete value until they are changed by a new material. The RGB values selected will be used under the alias `<X> Color` while the alpha component will be under `<X> Color Alpha`, where <X> representing the color registers name. All colors are constant rgba colors, though some registers have alternative usages or ad hoc purposes beyond being a free use color.
+
+* Environment Color: Free use register
+
+* Primitive Color: Free use register. Encoded at the same time as LOD variables but those have no impact on the color.
```

Comment:
I'd get rid of the `Encoded at the same time as LOD variables but those have no impact on the color.` part and just say something like `Primitive Color also has some additional capabilities, but these are not relevant to F3D materials in Blender.` Besides the LOD variables, prim color is buffered so that it can be changed between primitives without needing any kind of syncs. This is the only register which has this property.